flume-ng-doc/sphinx/FlumeUserGuide.rst

```diff
@@ -1375,6 +1375,7 @@ skipToEnd                           false                          Whether to sk
 idleTimeout                         120000                         Time (ms) to close inactive files. If the closed file is appended new lines to, this source will automatically re-open it.
 writePosInterval                    3000                           Interval time (ms) to write the last position of each file on the position file.
 batchSize                           100                            Max number of lines to read and send to the channel at a time. Using the default is usually fine.
+maxBatchCount                       Long.MAX_VALUE                 Controls the number of batches being read consecutively from the same file. It can be used to prevent reading a file in endless loop when the file is written by high frequency.
```

Comment:
I would add a little bit more explanation. for example:
If the source is tailing multiple files and one of them is written at a fast rate, it can prevent other files to be read. In this case lower this value.

Reply:
Modified.